convert2rhel/checks.py

```diff
@@ -83,6 +84,22 @@ def perform_pre_ponr_checks():
     ensure_compatibility_of_kmods()
 
 
+def check_custom_repos_are_valid():
+    """To prevent failures past the PONR, make sure that the enabled custom repositories are valid.
+
+    What is meant by valid:
+    - YUM/DNF is able to find the repoids (to rule out a typo)
+    - the repository "baseurl" is accessible and contains repository metadata
+    """
+    logger.task("Prepare: Checking if custom repositories are accessible")
```

Comment:
Better to say we're checking `--enablerepo` explicitly than "custom repos", as it can mean pretty much all repos including those we don't verify

```suggestion
    logger.task("Prepare: Checking if --enablerepo repositories are accessible")
```

Reply:
Yeah, this is better.
I initially thought it's not enough accurate because one can use --enablerepo also with RHSM to override the default RHSM repos we enable based on our config settings, and we do not validate these overriding RHSM repos, but we don't need to explain 100% everything in the `task` level log message :)